spec/numberformat.html

```diff
@@ -1599,20 +1613,20 @@ <h1>Runtime Semantics: StringIntlMV</h1>
           1. Let _b_ be 0.
           1. Let _n_ be 0.
         1. If |ExponentPart| is present, let _e_ be MV of |ExponentPart|. Otherwise, let _e_ be 0.
-        1. Return (_a_ + (_b_ × 10<sup>-_n_</sup>)) × 10<sup>_e_</sup>.
+        1. Return « (_a_ + (_b_ × 10<sup>-_n_</sup>)) × 10<sup>_e_</sup>, _n_ - _e_ ».
```

Comment:
Thought: I was expecting us to carry significant digits rather than fraction digits... but the two are interchangeable. It's just weird to have negative fraction digits being carried around.

Reply:
I've now replaced [[FractionDigits]] with [[StringDigits]].